crates/csslsrs/tests/document_symbols.rs

```diff
@@ -0,0 +1,608 @@
+#![allow(deprecated)]
```

Comment:
would love a test for a at rule with nested properties like 

```css
@font-palette-values --my-palette {
  font-family: Bixa;
  base-palette: 1;
  override-colors: 0 #ff0000;
}
```

Reply:
Added. Tell me if the result suit you.